src/jupyter_contrib_nbextensions/nbconvert_support/embedhtml.py

```diff
@@ -49,29 +53,37 @@ def replfunc(self, match):
             with open(filename, 'rb') as f:
                 data = f.read()
 
-        self.log.info("embedding url: %s, format: %s" % (url, imgformat))
```

Comment:
I think it is helpful to have a log of the embedded images. Better move it to `self.log.debug`